[BUG] make verify-* targets should ignore non-microshift go files

#### What happened:
make verify-govet fails when any go file specifies unimportable paths.  It's useful to specify these tools as go imports in a non-project go file in order to force go mod to treat them as versioned dependencies.

#### What you expected to happen:
make verify-govet should not process code outside the microshift code base.

#### How to reproduce it (as minimally and precisely as possible):

1. add a file, preferably outside the code base e.g. to ./scripts, that specifies a build tool as an import. i.e. `./scripts/tools`:
```
// This package imports things required by build scripts, to force `go mod` to see them as dependencies
package tools

import (
        _ "github.com/go-bindata/go-bindata/go-bindata"
        _ "k8s.io/kube-openapi/cmd/openapi-gen"
)
```
2. `$ make verify`

#### Relevant Logs 
```
❯ make verify
Running `gofmt -s -l` on 53 file(s).
go vet -mod=vendor ./...
scripts/tools.go:5:2: import "github.com/go-bindata/go-bindata/go-bindata" is a program, not an importable package
scripts/tools.go:6:2: import "k8s.io/kube-openapi/cmd/openapi-gen" is a program, not an importable package
make: *** [verify-govet] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -20,6 +20,9 @@ IMAGE_REPO :=quay.io/microshift/microshift
 OUTPUT_DIR :=_output
 CROSS_BUILD_BINDIR :=$(OUTPUT_DIR)/bin
 
+# restrict included verify-* targets to only process project files
+GO_PACKAGES=$(go list ./cmd/... ./pkg/...)
+
 GO_EXT_LD_FLAGS :=-extldflags '-static'
 GO_LD_FLAGS :=-ldflags "-X k8s.io/component-base/version.gitMajor=1 \
                    -X k8s.io/component-base/version.gitMinor=20 \

